src/osmesa_init.c

```diff
@@ -0,0 +1,35 @@
+
+#include "internal.h"
+
+//////////////////////////////////////////////////////////////////////////
+//////                       GLFW internal API                      //////
+//////////////////////////////////////////////////////////////////////////
+
+
+//////////////////////////////////////////////////////////////////////////
+//////                       GLFW platform API                      //////
+//////////////////////////////////////////////////////////////////////////
+
+int _glfwPlatformInit(void)
+{
+  if (!_glfwInitThreadLocalStoragePOSIX()) {
```

Comment:
GLFW uses [Allman style](https://en.wikipedia.org/wiki/Allman_style) for code.

Indentation is four spaces.